compiler-plugin-test/src/test/java/io/ballerina/stdlib/persist/compiler/script/CompilerPluginTest.java

```diff
@@ -0,0 +1,348 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.persist.compiler.script;
+
+import io.ballerina.projects.DiagnosticResult;
+import io.ballerina.projects.Package;
+import io.ballerina.projects.ProjectEnvironmentBuilder;
+import io.ballerina.projects.directory.BuildProject;
+import io.ballerina.projects.environment.Environment;
+import io.ballerina.projects.environment.EnvironmentBuilder;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Tests for sql script generator.
+ */
+public class CompilerPluginTest {
+
+    private static ProjectEnvironmentBuilder getEnvironmentBuilder() {
+        Path distributionPath = Paths.get("../", "target", "ballerina-runtime")
+                .toAbsolutePath();
+        Environment environment = EnvironmentBuilder.getBuilder().setBallerinaHome(distributionPath).build();
+        return ProjectEnvironmentBuilder.getBuilder(environment);
+    }
+
+    private Package loadPackage(String path) {
+        Path projectDirPath = Paths.get("src", "test", "resources", "test-src", "sql_script").
+                toAbsolutePath().resolve(path);
+        BuildProject project = BuildProject.load(getEnvironmentBuilder(), projectDirPath);
+        return project.currentPackage();
+    }
+
+    @Test
+    public void testGenerateSqlScript() throws IOException {
+        String fileContent = "DROP TABLE IF EXISTS Medical_Need;\n" +
+                "\n" +
+                "DROP TABLE IF EXISTS Item;\n" +
+                "CREATE TABLE Item ( \n" +
+                "\tid INT NOT NULL AUTO_INCREMENT, \n" +
+                "\tname VARCHAR(191) NOT NULL,\n" +
+                "\tPRIMARY KEY(id) \n" +
+                ")  AUTO_INCREMENT = 15;\n" +
```

Comment:
Can we add a `\t` before AUTO_INCREMENT like
 ```suggestion
                ")\tAUTO_INCREMENT = 15;\n" +
```